package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "imagekit-javascript",
-  "version": "3.1.0",
+  "version": "4.0.0",
   "lockfileVersion": 1,
```

Comment:
this should be upgraded. all tooling should be upgraded so that anyone who installs this package don't get such warnings.